src/components/RunGraph.vue

```diff
@@ -41,6 +44,10 @@
     },
   })
 
+  watch(() => props.selected, selected => selectNode(selected))
```

Comment:
you're ok with this firing twice for each selection because of the escape hatch in `selectNode`, right?

Reply:
Correct, generally I'd have that check here but wanted that to happen no matter what triggered it.

Reply:
makes sense